README.md

```diff
@@ -7,13 +7,19 @@ Go-tracing provides an easy way to use zipkin tracing with only four lines of co
 ## Quick start
 
 ```golang
+// import the library
+import "github.com/ricardo-ch/go-tracing"
+
 // set your tracer
-tracing.SetGlobalTracer(appName, "{zipkin_host}")
+tracing.SetGlobalTracer(appName, "{zipkin_url}")
 defer tracing.FlushCollector()
 
 // define a trace
 span, ctx := tracing.CreateSpan(ctx, "{span_name}", nil)
 defer span.Finish()
+
+// propagate the context downstream, for example, in an http.Handler wrapper:
+wrappedHttpHandler.ServeHTTP(responseWriter, httpRequest.WithContext(ctx))
```

Comment:
may be too much
theses last 2 lines should not be part of quick start
there is plenty of example in examples folder that will show theses kind of use